### ISSUE TITLE
Rename description "Remote universal OpenJDK service"

### ISSUE THREAD

**Issue by emilianbold** [foojay2020/nbplugin2#52]
I think users will be confused looking at this:

<img width="712" alt="Screenshot 2021-03-31 at 19 02 04" src="https://user-images.githubusercontent.com/991554/113393280-1bc65d00-939f-11eb-8823-d7a47980591d.png">

since they already have:

* Java Standard Edition
* Remote Java Standard Edition
* Remote Universal OpenJDK Service

Now, I think all of these have wrong makes but just looking at "Remote Universal OpenJDK Service" it's no obvious at all this is a way to download locally an OpenJDK build. Especially since it's called 'Remote xxx', ie. perhaps it has something to do with the other 'Remote Java Standard Edition' thing?

Not sure what a proper name would be, but something like "Download and install OpenJDK"?

**Comment by geertjanw** [comment]
I agree. Maybe your label makes more sense, I'd agree with it if you push that label.
